styles/vikunja/catppuccin.user.css

```diff
@@ -0,0 +1,282 @@
+==/UserStyle== */
+@-moz-document domain('vikunja.guaxinim.online') {
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+    
+    --site-background: @base;
+    --content-heading-color: @text;
+    --card-header-color: @text;
+    --card-color: @text;
+    --body-color: @text;
+    --label-color: @text;
+    --strong-color: @text;
+    --primary: @accent-color;
+    --primary-invert: @base;
+    --switch-view-color: @base;
+    --input-background-color: @base;
+    --input-icon-color: @text;
+    --table-row-hover-background-color: @mantle;
+    --table-head-cell-color: @text;
+    --input-border-color: @overlay0;
+    --link-hover: @accent-color;
+    --white: @base;
+    --grey: @overlay0;
+    --card-content-background-color: @base;
+    --success: @green;
+    --danger: @red;
+    --warning: @yellow;
+
+    body,
+    div#app,
+    header,
+    aside.menu-container,
+    footer.card-footer {
+      background: @base !important;
+    }
+    
+    svg.llama-cool > circle,
+    svg.logo > path[fill="#196aff"] {
+      fill: @accent-color;
+    }
+    
+    svg.logo,
+    h1 {
+      color: @text;
+    }
+    
+    h1.title.task-id,
+    div.detail-title,
+    div.detail-title span {
+      color: @overlay0;
+    }
+    
+    button {
+      box-shadow: none !important;
+    }
+    
+    button.is-danger {
+      color: @red !important;
+      background: none;
+      
+      &:hover,
+      &:active {
+        background: @red !important;
```

Comment:
could we make the hover/active provide more feedback either with `lighten()`, `darken()` or similar.